downstream/titles/eda/eda-getting-started-guide/docinfo.xml

```diff
@@ -1,4 +1,4 @@
-<title>Getting Started with Event-Driven Ansible Guide</title>
+<title>Getting started with event-driven Ansible guide</title>
```

Comment:
Event-Driven Ansible is a proper noun and should be capitalized

Getting Started with Event-Driven Ansible guide